spec/models/user_spec.rb

```diff
@@ -3,7 +3,6 @@
 RSpec.describe User, type: :model do
   describe 'validations' do
     it { should validate_presence_of :first_name }
-    it { should validate_presence_of :last_name }
     it { should validate_presence_of :email }
```

Comment:
Is there a reason why we're getting rid of the last_name validation?

Reply:
Based on our check-in today, it's fine with me if this stays as `first_name`, but I'm also open to moving it to the whole name instead (`name`).